test-framework/sudo-compliance-tests/src/sudo/apparmor.rs

```diff
@@ -0,0 +1,39 @@
+use sudo_test::{Command, Env};
+
+use crate::{Result, PANIC_EXIT_CODE};
+
+#[cfg(feature = "apparmor")]
+#[test]
```

Comment:
```suggestion
//TODO: this file needs to be present in the *host* machine.
//This can't be done automatically in the test framework. See issue #1094 
#[test]
```

Reply:
I've created issue #1094 to track this.